pageserver/src/tenant/remote_timeline_client.rs

```diff
@@ -931,26 +922,22 @@ impl RemoteTimelineClient {
 
         let index_file_path = timeline_storage_path.join(Path::new(IndexPart::FILE_NAME));
 
-        debug!("deleting index part");
+        debug!("enqueuing index part deletion");
+        self.deletion_queue_client
+            .push_immediate([index_file_path].to_vec())
```

Comment:
```suggestion
            .push_immediate(vec![index_file_path])
```

Reply:
I guess it just looked weird to me when skimming through this. I do wonder why does it need to be a vec at all. I'll check this...

Reply:
The whole vec is sent away in a message, so accepting Vec makes sense. Looked around, no real benefit for one over the other. In more recently edited code `vec!` is more popular.

Resolving..